packages/devtools/lib/src/timeline/frames_bar_chart.dart

```diff
@@ -9,11 +9,19 @@ import '../ui/elements.dart';
 import '../ui/fake_flutter/dart_ui/dart_ui.dart';
 import '../ui/flutter_html_shim.dart';
 import '../utils.dart';
+import 'package:mutex/mutex.dart';
 import 'timeline.dart';
 import 'timeline_controller.dart';
 import 'timeline_protocol.dart';
+import 'frames_bar_plotly.dart';
+
+import '../ui/plotly.dart';
+
+import 'dart:developer';
```

Comment:
remove unused imports here and elsewhere

Reply:
Debugging stuff removed - Done.